packages/swapkit/helpers/src/modules/swapKitError.ts

```diff
@@ -47,7 +47,8 @@ const errorMessages = {
   core_transaction_deposit_insufficient_funds_error: 10311,
   core_transaction_deposit_gas_error: 10312,
   core_transaction_invalid_sender_address: 10313,
-  core_transaction_deposit_server_error: 10313,
+  core_transaction_deposit_server_error: 10314,
```

Comment:
I guess the double number usage caused from 2 merged PRs? 
Here was the duplication introduced:
https://github.com/thorswap/SwapKit/pull/428/files#diff-68f6c81e60089ae998fae74182d1f3ae11c416a989e2dc6774074ecf00fb5313R48